template-for-repository/proofs/Makefile-project-defines

```diff
@@ -20,6 +20,11 @@
 # LINK_FLAGS =
 
 # Preprocessor include paths -I...
+# It is likely that you will want to add at least
+#     INCLUDES += -I$(HELPER_DIR)/include
+# but you will want to decide what order that comes in relative to the
+# other includes in your project.
+#
```

Comment:
I'd shorten this to "Consider adding INCLUDES += ..."

Reply:
Done